projects/cppcheck/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build fuzzer
+
+cd fuzzer-cli
+make oss-fuzz-client
+cp oss-fuzz-client $OUT/
+
```

Comment:
at some point, you should add a seed corpus and dictionary for fuzzing efficiency.

Reply:
For information..

the client takes the binary input from the fuzzer and conerts the ones and zeros it get into code.

For instance imagine that the fuzzer generates 2 bytes "AA". That is translated into this code:

    void f() {
        x=( ){ } >= %;
    }

"inferno" is translated to:

    void f() {
        x= return int enum int typedef && return continue >= ^ <;
    }

It's not 100% valid but we can tweak that...

Reply:
Sounds good, thanks!